themes/sdss_subtheme/templates/components/news-list/news-list.ui_patterns.yml

```diff
@@ -31,12 +36,17 @@ sdss-news-list:
       type: text
       label: "Source"
       description: "The source field."
-      preview: "I am a link"
+      preview: "I am text"
     news_url:
       type: text
       label: "URL for item"
       description: "URL for image and heading"
       preview: "http://www.stanford.edu/"
+    su_news_source:
+      type: text
+      label: "External URL for item"
+      description: "URL for image and heading"
+      preview: "http://www.google.com/"
```

Comment:
`su_news_source` gets confusing fast when working in the templates. Would we want to rename this one to `news_external_url`? That would follow the conventions in the rest of the component and be easier to understand.